Doc/library/stdtypes.rst

```diff
@@ -478,6 +478,24 @@ class`. In addition, it provides a few more methods:
 
     .. versionadded:: 3.1
 
+.. method:: int.bit_count()
+
+    Return the number of ones in the binary representation of the absolute
```

Comment:
To aid searches, I'm wondering whether it's also worth mentioning the terms "population count" and "Hamming weight" in this docstring.

Reply:
Added "population count", following Victor's suggestion. That's probably good enough.